hugo/layouts/partials/header.html

```diff
@@ -1,30 +1,72 @@
-<nav class="fullWidth topbar {{ .Params.topbar }}" id="topBar">
+<nav class="topbar" id="topbar">
     <div>
         <a href="{{ .Site.BaseURL }}" alt="source{d}">
-            <div class="logo"></div>
+            <div class="logo"><img src="{{ .Site.BaseURL }}/img/logos/source-logo-white.svg" alt="source{d}" /></div>
```

Comment:
Sergio discovered that it's better to use [absURL](https://gohugo.io/templates/functions#absurl-relurl)